torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -550,9 +550,35 @@ RegisterOperators reg2({
     DEFINE_BINARY_OP(aten::add, a + b)
     DEFINE_BINARY_OP(aten::sub, a - b)
     DEFINE_BINARY_OP(aten::mul, a * b)
-    DEFINE_BINARY_OP(aten::div, a / b)
     DEFINE_BINARY_OP(aten::pow, static_cast<decltype(a)>(pow(a, b)))
 
+    // NB: Division differs in py2 / py3
+    // - py2: (int, int) -> int
+    // - py3: (int, int) -> float
+    // XXX: These should probably not be the same op as aten::div
+#if PY_MAJOR_VERSION > 2
```

Comment:
This file really shouldn't depend on the Python version, because it is part of the no-python build too (in which case we'll choose the Python 2 behavior!).